src/Hazelcast.Net.Linq.Async/Visitors/QueryFormatter.cs

```diff
@@ -44,18 +44,18 @@ internal QueryFormatter()
         /// </summary>
         /// <param name="expression">tree</param>
         /// <returns>(SQL statement, Value of variables)</returns>
-        public static (string, IEnumerable<object>) Format(Expression expression)
+        public static (string, object[]) Format(Expression expression)
         {
             var f = new QueryFormatter();
             f.Visit(expression);
-            return (f.ToString(), f._values);
+            return (f.ToString(), f._values.ToArray());
```

Comment:
It's usually more efficient to `ToList`, do we need `ToArray`?

Reply:
`_values` is already a List but I need a copy of the params, otherwise `SqlService` tries to serialize the list as whole instead of its params.